### DISCUSSION TITLE
Fedora 30 than Fedora 39 is better!!!

### DISCUSSION THREAD

**Discussion by hangshan-boop** [community/community#121525] [Programming Help]
### Select Topic Area

Question

### Body

  I use QQ in Fedora 39 that is bad. QQ can't send pdf and picture.If only these, I can be endure. Firefox is bad as the same.Loading web pages take long time. My graduation thesis can't be submitted; my literature looks difficult. Fedora more and more bad. 
  I hope that the redhat management should reflect

**Comment by ROBERT-MCDOWELL** [comment]
this has not any link with linux fedora but the sofware you are using

**Comment by hangshan-boop** [comment]
Why can't it be compatible with the Firefox kernel？We need to consider user.&nbsp;




---Original---
From: "ROBERT ***@***.***&gt;
Date: Mon, Apr 29, 2024 19:29 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Fedora 30 than Fedora 39 is better!!!(Discussion #121525)




 
this has not any link with linux fedora but the sofware you are using

**Comment by hangshan-boop** [comment]
We consider safety. We can give up internet. we can install software, then release.



---Original---
From: "ROBERT ***@***.***&gt;
Date: Mon, Apr 29, 2024 19:29 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Fedora 30 than Fedora 39 is better!!!(Discussion #121525)




 
this has not any link with linux fedora but the sofware you are using

**Comment by hangshan-boop** [comment]
I don't understand Fedora 39 and 40.



---Original---
From: "ROBERT ***@***.***&gt;
Date: Mon, Apr 29, 2024 19:29 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Fedora 30 than Fedora 39 is better!!!(Discussion #121525)




 
this has not any link with linux fedora but the sofware you are using

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @hangshan-boop !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Comment by ebndev** [comment]
Hi @hangshan-boop,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this, such as the Firefox support. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
